shlib/server.sh

```diff
@@ -209,7 +209,6 @@ calculate_server_args() {
     fi
   fi
 
-  local server_cert_path server_ca_path server_client_cert_path
```

Comment:
I think I see what you are doing here, can you add a comment to the signature of this function saying which global variables it sets?